### PR TITLE
Issue 3 user can view store listing for game deals

### PR DESCRIPTION
**Pull request by FrankBishop**
Screenshot (231)
Screenshot (230)

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -2,13 +2,29 @@ var searchForm = document.querySelector('.search-form');
...
+var searchResults = document.querySelector('.search-results');
+var storeListings = document.querySelector('.store-listings-results')
+
+var storesList = [];
 
 
 searchForm.addEventListener('submit', submitAction);
 
+function stores() {
+  var stores = new XMLHttpRequest();
```
Comment (codingcodymiller) [inline js/main.js]: You should change the name of this variable, you don't want variables with the same name as the function they are in

File: js/main.js [hunk 2]
```diff
@@ -52,7 +65,65 @@ function getResults(searchRequest) {
...
+      storeIcon.setAttribute('src', iconImage);
+      storeIcon.className = 'list-image picture-column';
+      priceResult.appendChild(storeIcon);
+      var storeName = document.createElement('h2');
+      var storeActualName;
+      for (var j = 0; j < storesList.length; j++) {
+        if (storeId === storesList[j].storeID) {
+          storeActualName = storesList[j].storeName;
+        }
+      }
```
Comment (codingcodymiller) [inline js/main.js]: Rather than looping over the store list twice, you should be able to do this in the above loop too 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by FrankBishop** [comment]
Thanks for the feedback, I made the requested changes.

File: js/main.js [hunk 3]
```diff
@@ -91,30 +91,26 @@ function buyNow(event) {
...
       var iconImage;
-      for(var k = 0; k < storesList.length; k++) {
-        if (storeId === storesList[k].storeID) {
-          iconImage = "https://www.cheapshark.com" + storesList[k].images['banner'];
+      for(var j = 0; j < storesList.length; j++) {
+        if (storeId === storesList[j].storeID) {
+          iconImage = "https://www.cheapshark.com" + storesList[j].images['banner'];
+        }
+        if (storeId === storesList[j].storeID) {
+          storeActualName = storesList[j].storeName;
```
Comment (codingcodymiller) [inline js/main.js]: You don't need two conditionals here, you can do the work for both of these conditionals in one conditional

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**